webrender/src/device/gfx/device.rs

```diff
@@ -1918,6 +1928,7 @@ impl<B: hal::Backend> Device<B> {
         filter: TextureFilter,
         render_target: Option<RenderTargetInfo>,
         layer_count: i32,
+        texture_usage: TextureUsage,
```

Comment:
Don't we already know if it's a render target based on the `render_target` parameter? Does a new parameter help?

Reply:
Yes we know that this is a render target from that parameter, but there is a TextureCache texture which is also a render target, but can persist across multiple frames and we can't use the linear allocation for that: 
https://github.com/szeged/webrender/blob/master/webrender/src/renderer.rs#L3462

Reply:
in this case, can we extend `RenderTargetInfo` ? perhaps, something as simple as a `persistent: bool` field would work best.